indigo_api/exporters.py

```diff
@@ -205,7 +210,11 @@ def insert_frontmatter(self, document):
         """ Generates the frontmatter and inserts it before the body (and coverpage if present) in the XML.
             The document's document_xml isn't updated.
         """
+        # explicitly set the document's language for e.g. numbered title to be translated
+        activate(document.django_language)
```

Comment:
rather user `override()` which ensures the deactivate is called if there is an error:

```python
with override(document.django_language):
  frontmatter = ...
```

Reply:
thanks!